.gitmodules

```diff
@@ -28,6 +28,7 @@
 [submodule "cmake/external/onnx-tensorrt"]
 	path = cmake/external/onnx-tensorrt
 	url = https://github.com/onnx/onnx-tensorrt.git
+        branch = v5.0
```

Comment:
Is this change needed ?

Reply:
onnx-tensorrt master branch is under big changes for new tensorrt release, so we point to a working branch before onnx-tensorrt changes are done.